src/resources/js/tickets-block.js

```diff
@@ -45,16 +45,46 @@ tribe.tickets.block = {
 		'.tribe-block__tickets__item__quantity__remove, .tribe-block__tickets__item__quantity__add',
 		function( e ) {
 			e.preventDefault();
-			var input  = $( this ).parent().find( 'input[type="number"]' );
-			var add    = $( this ).hasClass( 'tribe-block__tickets__item__quantity__add' );
+			var input = $( this ).parent().find( 'input[type="number"]' );
+			var add = $( this ).hasClass( 'tribe-block__tickets__item__quantity__add' );
+			var step = ( undefined !== Number( input[ 0 ].step ) ) ? Number( input[ 0 ].step ) : 1;
+			var originalValue = Number( input[ 0 ].value );
 
-			// stepUp or stepDown the input according to the button that was clicked
-			add ? input[0].stepUp() : input[0].stepDown();
 
-			// Trigger the on Change for the input as it's not handled via stepUp() || stepDown()
-			input.trigger( 'change' );
+			// stepUp or stepDown the input according to the button that was clicked
+			// handle IE/Edge
+			if ( add ) {
+				var max = ( undefined !== Number( input[ 0 ].max ) ) ? Number( input[ 0 ].max ) : 0;
+
+				if ( typeof input[ 0 ].stepUp === 'function' ) {
+					try {
+						input[ 0 ].stepUp();
+					} catch ( ex ) {
+						input[ 0 ].value = ( 0 === max || max > input[ 0 ].value ) ? originalValue + step : max;
+					}
+				} else {
+					input[ 0 ].value = ( 0 === max || max > input[ 0 ].value ) ? originalValue + step : max;
+				}
+			} else {
+				var min = ( undefined !== Number( input[ 0 ].min ) ) ? Number( input[ 0 ].min ) : 0;
+
+				if ( typeof input[ 0 ].stepDown === 'function' ) {
+					try {
+						input[ 0 ].stepDown();
+					} catch ( ex ) {
+						input[ 0 ].value = ( min < input[ 0 ].value ) ? originalValue - step : min;
```

Comment:
in the case `step > 1`:

```suggestion
						input[ 0 ].value = ( min < originalValue - step ) ? originalValue - step : min;
```